module.nix

```diff
@@ -351,6 +374,33 @@ in
       }
     ))
 
+    # RAC outpost
+    (mkIf config.services.authentik-rac.enable (
+      let
+        cfg = config.services.authentik-rac;
+      in
+      {
+        systemd.services.authentik-rac = {
+          wantedBy = [ "multi-user.target" ];
+          wants = [ "network-online.target" ];
+          after = [
+            "network-online.target"
+            "authentik.service"
+          ];
+          serviceConfig = {
+            RuntimeDirectory = "authentik-rac";
+            UMask = "0027";
+            WorkingDirectory = "%t/authentik-rac";
+            DynamicUser = true;
+            ExecStart = "${config.services.authentik.authentikComponents.gopkgs}/bin/rac";
+            EnvironmentFile = mkIf (cfg.environmentFile != null) [ cfg.environmentFile ];
+            Restart = "on-failure";
+            BindReadOnlyPaths = "${lib.getExe pkgs.guacamole-server}:/opt/guacamole/sbin/guacd";
```

Comment:
I think it's more reasonable to just patch the path int he source code given we're compiling the stuff ourselves already.
We have quite a large closure already, so I'd just do it by default it and see if somebody complains.
This has the side-effect this is also usable outside of the unit.

Reply:
You can already run it outside the unit with `systemd-run`, plus it just seems a bit more effort to create and maintain a patch for an exceedingly unlikely case.

But I do not feel strongly about this, if you prefer I'll make it a patch.